yarn.lock

```diff
@@ -1749,9 +1749,9 @@ aws-sign2@~0.7.0:
   integrity sha1-tG6JCTSpWR8tL2+G1+ap8bP+dqg=
 
 aws4@^1.8.0:
-  version "1.8.0"
-  resolved "https://registry.yarnpkg.com/aws4/-/aws4-1.8.0.tgz#f0e003d9ca9e7f59c7a508945d7b2ef9a04a542f"
-  integrity sha512-ReZxvNHIOv88FlT7rxcXIIC0fPt4KZqZbOlivyWtXLt8ESx84zd3kMC6iK5jVeS2qt+g7ftS7ye4fi06X5rtRQ==
+  version "1.9.0"
```

Comment:
any reason this was updated and should we update the package.json then to account for the update.

Reply:
I attached the reason here https://github.com/airswap/airswap-protocols/pull/352/files#r355651146
These don't affect the other package versions unless we want to update those with the most recent version.